src/services/lock/service/lock.service.ts

```diff
@@ -118,131 +102,35 @@ export class LockService {
     });
 
     this.#logger.log(`This node is currently registered as lock keeper for ${id}`);
-
     this.registeredLocks[id] = {};
 
     return lock;
   }
 
-  /**
-   * Unregisters a previously acquired lock, making it available for other instances.
-   * @param {string} id - The unique identifier of the lock to be unregistered.
-   */
   async unregisterLock(id: string): Promise<void> {
     const lock = await this.lockRepository.findOneById(id);
 
-    // Check if the lock exists and is registered to the current instance
     if (lock && lock.registered_id === this.identity.id) {
-      // Option 1: Set the registered_id to null, indicating the lock is free
       await this.lockRepository.findOneByIdAndMakeInactiveIfExists({ id });
-
-      // Option 2: Delete the lock from the repository if locks are treated as ephemeral
-      // await this.lockRepository.deleteLock(id);
```

Comment:
Should this have been commented out? Looks important.

Reply:
we mark this node as inactive on line 137, meaning other nodes can register themselves. Deleting and marking things as inactive serve the same purpose here